app/Services/API/Customer/CustomerService.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Services\API\Customer;
+
+use App\Models\Customer;
+
+class CustomerService
```

Comment:
Change this to `CustomerRepository` under `Repositories` namespace
later we may have real services